pkg/ebpf/tracee.go

```diff
@@ -1287,3 +1274,32 @@ func (t *Tracee) updateKallsyms() {
 		t.kernelSymbols = kernelSymbols
 	}
 }
+
+// GetCaptureEventsConfig set events used to capture data
+func GetCaptureEventsConfig(cfg *Config) map[int32]eventConfig {
```

Comment:
For code readability (knowing which events are chosen), I think it would be better to put this function near CreateEssentialEventsList. Even better - keep this code in one part (combine CreateEssentialEventsList and GetCaptureEventsConfig) as done at https://github.com/aquasecurity/tracee/pull/1791/files#diff-607a60afe9e4f5447f69749836213a1ee925bd27961145aa924c50a037f58a85R211

Reply:
This is resolved.